### PR TITLE
Garnish

### PR DESCRIPTION
**Pull request by brisvag**
Renamed project to GARNISH!

### CODE REVIEW

**Comment by brisvag** [comment]
I fixed all the mess I made. Also, changed from decorator to wrapper for the `extension` function, so it's still possible to call `extend_garnish` from a python script.

Should be mergeable!

File: README.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
-# **SK**etch **E**fficiently a **WE**b of **R**elationships
-SKEWER allows a coarse grained structure to be visualized in pymol like an atomistic structure by drawing bonds and elastic network.
+# Garnish Accurately Represents a Network-Interconnected System in a Hearbeat
```
Comment (mjtadema) [inline README.md]: Hey awesome you can add comments to lines!
Anyway you missed a t in heartbeat :P
```suggestion
# Garnish Accurately Represents a Network-Interconnected System in a Heartbeat
```

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import setuptools
+
+with open("README.md",'r') as f:
+    long_description = f.read()
+
+setuptools.setup(
+        name="martini-garnish",
+        version="0.2-alpha",
+        author=["Matthijs Tadema", "Lorenzo Gaifas"],
+        author_email="M.J.Tadema@protonmail.com",
```
Comment (mjtadema) [inline setup.py]: Please add your email!

Comment (brisvag) [reply setup.py]: done already, hopefully in a list it works ok!

**Pull request merged**